libfeed/feed-store-provider.c

```diff
@@ -137,7 +137,10 @@ remove_uri_prefix (const gchar *uri)
   g_autoptr(SoupURI) soup_uri = soup_uri_new (uri);
   const gchar *path = soup_uri_get_path (soup_uri);
```

Comment:
It looks to me that this would not fix Roddy's crash; at least to a first extent, the assertion failure is hit because `soup_uri` is `NULL` and is passed to `soup_uri_get_path()`.

Reply:
Hrm, yes that is right. Updated.